dev_docs/key_concepts/api_authorization.mdx

```diff
@@ -0,0 +1,306 @@
+- [Authorization Response in Route Handlers](#authorization-response-available-in-route-handlers)
+- [OpenAPI Specification (OAS) documentation](#openapi-specification-oas-documentation)
+- [Migrating from `access` Tags to `security` Configuration](#migrating-from-access-tags-to-security-configuration)
+
+## API authorization
+Kibana API routes do not have any authorization checks applied by default. This means that your APIs are accessible to anyone with valid credentials, regardless of their permissions. This includes users with no roles assigned.
+This on its own is insufficient, and care must be taken to ensure that only authorized users can invoke your endpoints.
+
+Kibana leverages <DocLink id="kibDevDocsSavedObjectsIntro" text="Saved Objects" /> for a majority of its persistence. The Saved Objects Service performs its own authorization checks, so if your API route is primarily a CRUD interface to Saved Objects, then your authorization needs are already met.
+This is also true for derivatives of the Saved Objects Service, such as the Alerting and Cases services.
+
+If your endpoint is not a CRUD interface to Saved Objects, then your route ensure that only authorized users can invoke your endpoint. This is **especially** important if your route does any of the following:
+1. Performs non-insignificant processing, causing load on the Kibana server.
+2. Calls Elasticsearch using the internal `kibana_system` user.
+3. Calls a third-party service.
+4. Returns any non-public information to the caller, such as system configuration or state.
+
+## [Deprecated] Adding API authorization with `access` tags
+`access` tags are used to restrict access to API routes. They are used to ensure that only users with the required privileges can access the route.
+
+Example configuration:
+```ts
+router.get({
+  path: '/api/path',
+  options: {
+    tags: ['access:<privilege_1>', 'access:<privilege_2>'],
+  },
+  ...
+}, handler);
+```
+
+More information on adding `access` tags to your routes can be found temporarily in the [legacy documentation](https://www.elastic.co/guide/en/kibana/current/development-security.html#development-plugin-feature-registration)
+
+### Why not add `access` tags to all routes by default?
+Each authorization check that we perform involves a round-trip to Elasticsearch, so they are not as cheap as we'd like. Therefore, we want to keep the number of authorization checks we perform within a single route to a minimum.
+Adding an `access` tag to routes that leverage the Saved Objects Service would be redundant in most cases, since the Saved Objects Service will be performing authorization anyway.
+
+
+## Adding API authorization with `security` configuration
+`KibanaRouteOptions` provides a security configuration at the route definition level, offering robust security configurations for both **Classic** and **Versioned** routes.
+
+### Key features of the new security configuration:
```

Comment:
This is currently "new", but I know that I'll personally forget to remove this once it's no longer new. Let's shorten the heading to keep this future-proof.

```suggestion
### Key features:
```